src/ex2/ItemManager.js

```diff
@@ -0,0 +1,25 @@
+class ItemManager {
+    constructor() {
+        this.taskArr = JSON.parse(localStorage.getItem("tasks")) || [];
+    }
+
+    addNewTask(taskValue) {
+        this.taskArr.push(taskValue); //add the user value to the task array
+        localStorage.setItem("tasks", JSON.stringify(this.taskArr));//add the updated task array to the localStorage
+    }
+
+    removeTask(taskValue) {
+        this.taskArr.splice(this.taskArr.indexOf(taskValue), 1);
+        localStorage.setItem("tasks", JSON.stringify(this.taskArr)); //add the updated task array to the localStorage
```

Comment:
you call this line a lot, I think it would have been nice to have a helper function that does this